src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1749,43 +1749,39 @@ class CreateTableStatementSegment(ansi.CreateTableStatementSegment):
         OneOf(
             # Columns and comment syntax:
             Sequence(
-                OneOf(
-                    Bracketed(
-                        Delimited(
-                            OneOf(
-                                Sequence(
-                                    Ref("ColumnReferenceSegment"),
-                                    Ref("DatatypeSegment"),
-                                    AnyNumberOf(
-                                        # A single COLLATE segment can come before or
-                                        # after constraint segments
-                                        OneOf(
-                                            Ref("ColumnConstraintSegment"),
-                                            Sequence(
-                                                "COLLATE",
-                                                Ref("CollationReferenceSegment"),
-                                            ),
+                Bracketed(
+                    Delimited(
+                        OneOf(
+                            Sequence(
+                                Ref("ColumnReferenceSegment"),
+                                Ref("DatatypeSegment"),
+                                AnyNumberOf(
+                                    # A single COLLATE segment can come before or
+                                    # after constraint segments
+                                    OneOf(
+                                        Ref("ColumnConstraintSegment"),
+                                        Sequence(
+                                            "COLLATE",
+                                            Ref("CollationReferenceSegment"),
                                         ),
                                     ),
                                 ),
-                                Ref("TableConstraintSegment"),
-                                Sequence(
-                                    "LIKE",
-                                    Ref("TableReferenceSegment"),
-                                    AnyNumberOf(
-                                        Ref("LikeOptionSegment"), optional=True
-                                    ),
-                                ),
                             ),
-                        )
-                    ),
-                    Bracketed(),  # Sometimes no columns, like if INHERITS
+                            Ref("TableConstraintSegment"),
+                            Sequence(
+                                "LIKE",
+                                Ref("TableReferenceSegment"),
+                                AnyNumberOf(Ref("LikeOptionSegment"), optional=True),
+                            ),
+                        ),
```

Comment:
sorry, i meant the optional to be part of the Delimited() block, inside the Bracketed